acmwebsite/model/meeting.py

```diff
@@ -1,18 +1,26 @@
-from sqlalchemy import *
-from sqlalchemy import Column, ForeignKey, Table
-from sqlalchemy.orm import mapper, relation
+import datetime
+
+from sqlalchemy import Column, ForeignKey
+from sqlalchemy.orm import relation
 from sqlalchemy.types import DateTime, Integer, Text, Unicode
 
-from acmwebsite.model import DBSession, DeclarativeBase, metadata
+from tg import config
+
+from acmwebsite.model import DeclarativeBase
 
 
 class Meeting(DeclarativeBase):
     __tablename__ = 'meeting'
 
     id = Column(Integer, primary_key=True)
     date = Column(DateTime, nullable=False)
+    duration = Column(Integer, nullable=True)
     location = Column(Text)
     title = Column(Unicode, nullable=False)
     description = Column(Unicode)
     survey_id = Column(Integer, ForeignKey('survey.id'), nullable=True, unique=True)
     survey = relation("Survey", back_populates="meeting")
+
+    def get_duration(self):
+        default_duration = int(config.get('meetings.default_duration'))
```

Comment:
I'm not sure how expensive this operation is, but it is pointless to waste CPU cycles on this if the duration is not `None`. It would be nice to short circuit this by doing:

    return datetime.timedelta(seconds=self.duration or
                                      int(config.get('meetings.default_duration')))

Or you could pull that out to a variable.